lib/output.js

```diff
@@ -146,11 +150,10 @@ function OutputStream(options) {
               case "\b": return "\\b";
               case "\f": return "\\f";
               case "\x0B": return options.screw_ie8 ? "\\v" : "\\x0B";
-              case "\u2028": return "\\u2028";
-              case "\u2029": return "\\u2029";
-              case "\ufeff": return "\\ufeff";
               case "\0":
                   return /[0-7]/.test(str.charAt(i+1)) ? "\\x00" : "\\0";
+              default:
+                  return to_ascii(s);
```

Comment:
I see. It appears that the original line `return s;` was never reached before your change.